packages/sections/src/variant/GWASCredibleSets/Body.tsx

```diff
@@ -0,0 +1,1534 @@
+import { Link, SectionItem, DataTable, ScientificNotation } from "ui";
+import { Box, Chip } from "@mui/material";
+import { naLabel, defaultRowsPerPageOptions } from "../../constants";
+import { definition } from ".";
+import Description from "./Description";
+
+function getColumns(id: string, label: string) {
+
+  return [
+    {
+      id: "leadVariant",
+      label: "Lead Variant",
+      renderCell: ({ variantId }) => (
+        variantId === id
+          ? <Box display="flex" alignContent="center" gap={0.5}>
+              <span>{variantId}</span>
+              <Chip label="self" variant="outlined" size="small"/>
+            </Box>
+          : <Link to={`/variant/${variantId}`}>{variantId}</Link>
+      ),
+      exportLabel: "Lead Variant",
+    },
+    {
+      id: "trait",
+      label: "Trait",
+      renderCell: d => (
+        <Link to={`/disease/${d["study.disease.id"]}`}>
+          {d["study.disease.name"]}
+        </Link>
+      ),
+      exportLabel: "Trait",
+    },
+    {
+      id: "study",
+      label: "Study",
+      renderCell: d => (
+        <Link external to={`https://www.ebi.ac.uk/gwas/studies/${d["study.id"]}`}>
+          {d["study.id"]}
+        </Link>
+      ),
+      exportLabel: "Study",
+    },
+    {
+      id: "pValue",
+      label: "P-Value",
+      comparator: (a, b) =>
+        a.pValueMantissa * 10 ** a.pValueExponent - b.pValueMantissa * 10 ** b.pValueExponent,
+      sortable: true,
+      renderCell: d => (
+        <ScientificNotation number={[d.pValueMantissa, d.pValueExponent]} />
+      ),
+      exportLabel: "P-Value",
+    },
+    {
+      id: "beta",
+      label: "Beta",
+      tooltip: "Beta with respect to the ALT allele",
+      renderCell: ({ beta }) => beta || beta === 0 ? beta.toPrecision(3) : naLabel,
+      exportLabel: "Beta",
+    },
+    // {
+    //   id: "ldr2",
+    //   label: "LD (r2)",
+    //   tooltip: "Linkage disequilibrium with the queried variant",
+    //   renderCell: ({ locus }) => (
+    //     locus.find(obj => obj.variantId === id).r2Overall.toFixed(2)
+    //   ),
+    //   exportLabel: "LD (r2)",
+    // },
```

Comment:
Remove commented code

Reply:
I think this column is likely to be added back in shortly.